content/applications/finance/fiscal_localizations/united_arab_emirates.rst

```diff
@@ -0,0 +1,222 @@
+--> Currencies`.
+
+It is also possible to use web services other than the **U.A.E Central Bank**, such as **xe.com** or
+the **European Central Bank**. To do so, click on the drop-down menu of the :guilabel:`Service`
+field and select one of the services.
+
+.. _uae/payroll:
+
+Payroll
+=======
+
+The :guilabel:`U.A.E. - Payroll` module includes all the rules configured under the
+:guilabel:`U.A.E. Employee Payroll Structure` in the **Salary Rules** section as per the U.A.E.
+rules and regulations. These rules are linked to their corresponding accounts in the **Chart of
+Accounts**.
+
+.. image:: united_arab_emirates/uae-localization-salary-rules.png
+   :align: center
+   :alt: The U.A.E. Employee Payroll Structure.
+
+Salary rules
+------------
+
+To apply these rules to an employee's contract, go to :menuselection:`Payroll --> Contracts -->
+Contracts` and select the contract's employee. In the :guilabel:`Salary Structure Type` field,
+select :guilabel:`UAE Employee`.
+
+.. image:: united_arab_emirates/uae-localization-salary-structure.png
+   :align: center
+   :alt: Select the Salary Structure Type to apply to the contract.
+
+Under the :guilabel:`Salary Information` tab, you can find details such as:
+
+- :guilabel:`Wage`;
+- :guilabel:`Housing Allowance`;
+- :guilabel:`Transportation Allowance`;
+- :guilabel:`Other Allowances`;
+- :guilabel:`Number of Days`: how the **End of Service Provision** is calculated.
+
+.. note::
+   **Leave deduction** is calculated using a salary rule linked to the unpaid leave time-off type;
+   however, any other deductions or reimbursements are done *manually* using other inputs.
+   Additionally, **overtime** is added *manually* through :menuselection:`Work Entries --> Work
+   Entries`, and **salary attachments** are generated from :menuselection:`Contracts --> Salary
+   Attachments`. :guilabel:`Create` an attachment and select the :guilabel:`Employee` and the
+   :guilabel:`Type (Attachment of Salary, Assignment of Salary, Child Support)`.
+
+.. tip::
+   If you wish to avoid a rule to appearing on a paycheck, go to :menuselection:`Payroll -->
+   Configuration --> Structures`. Click on the **U.A.E. structure**, select the rule to hide, and
+   uncheck :guilabel:`Appears on Payslip`.
+
+End of Service Provision
+------------------------
+
+The provision is defined as the total monthly allowance *divided* by 30 and then *multiplied* by the
+number of days set in the field :guilabel:`Number of days` at the bottom of a contract's form.
+
+The provision is then calculated via a salary rule associated with two accounts: the **End Of
+Service Indemnity (Expense account)** and the **End of Service Provision (Non-current Liabilities
+account)**. The latter is used to pay off the **End of service** amount by settling it with the
+**Payables account**.
+
+.. note::
+   The **End of service** calculations is based on the gross salary according to the United Arab
```

Comment:
I would suggest: The **End of service** amount [or other word more suitable] is calculated based on the employee's gross salary (according to the United Arab Emirates' HR laws), as well as the contract's start and end dates.

(Is the "according to the United Arab Emirates' HR laws" necessary here since we're talking about the UAE localization since the beginning?)